### ISSUE TITLE
Top task pages all need <title> tags

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9023]
## To Reproduce
1. Go to https://www.va.gov/eastern-oklahoma-health-care/register-for-care/
2. Note there is no <title> 

## Expected behavior
All top task content types should have a <title> that follow the pattern of
`<title>Make An Appointment | VA Eastern Oklahoma Health Care | Veterans Affairs</title>`
   
`<title>{node title} | {system name} | Veterans Affairs</title>`

## Additional context
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`
